projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+
+#### Automated Testing 
+
+The project is tested by the two automated test suites which have been developed by the W3C CCG for testing credential issuance (https://github.com/w3c-ccg/vc-api-issuer-test-suite) and credential verification (https://github.com/w3c-ccg/vc-api-verifier-test-suite).
+
+The project is also tested by unit tests and end-to-end tests. 
+
+Integration tests are implemented based on Energy and Mobility use cases, which have historically come from Elia and BloXmove energy and mobility as use-case requirements respectively. 
+
+### Documentation 
+
+The project contains technical documentation and tutorials to facilitate understanding of the API and experiment with the implementation: https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api/docs/tutorials 
+
+Furthermore, Energy Web has documentation on VC-API and general documentation on SSI available the Energy Web gitbook: 
+
+- https://energy-web-foundation.gitbook.io/energy-web/foundational-concepts/self-sovereign-identity-introduction
+- https://energy-web-foundation.gitbook.io/energy-web/ew-dos-technology-components-2023/identity-and-access-management-iam/iam-guides/verifiable-credential-api
+
+# Alignment with the OpenWallet Foundation Mission
+_Include a statement on alignment with the [OpenWallet Foundation mission](https://openwallet-foundation.github.io/tac/governance/charter/)._
+
+The project team believes that the project is well aligned with the OpenWallet Foundation mission. 
+
+An implementation of VC API developed and hosted by the OWF would advance the adoption of interoperable and privacy enhancing digital wallets. This is because VC API is an open specification of an interface for Verifiable Credential operations, which are relevant functions for digital wallets.
+Furthermore, as an HTTP API, the project is a technology well understood by many organizations and will plausibly accelerate their efforts to develop or integrate digital wallets. 
+
+In addition, development of a VC API implementation would promote the development and proliferation of the VC API specification itself, which is an open standard related to digital wallets. 
+
+The project has been open source since its inception and aims to be inclusive, in so far as it should be usable across a wide range of domains and infrastructure requirements.
+
+# Current Code of Conduct
+_Has this project adopted a Code of Conduct? If so, please include a link to it._
+
+https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/CODE_OF_CONDUCT.md
+
+# TAC Sponsor
+_Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
+
+The project has not yet identified a TAC sponsor.
+
+# Project License
+_Include the proposed project license (Apache 2.0 by default)._
+
+GNU General Public License 3.0 -https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/LICENSE 
```

Comment:
Since we have not had any projects submitted under this license, we will need to reach out to the Governing Board regarding proposed license.

From the [OWF Project Lifecycle | Labs Acceptance Criteria](https://tac.openwallet.foundation/governance/project-lifecycle/#labs)
> Document an intellectual property policy that leverages the Apache 2.0 license or an open license that has been approved by the OpenWallet Foundation's Governing Board.

Reply:
Thanks for the comment @tkuhrt . Do you know if there are any licenses other than Apache 2.0 that have already been approved by the OpenWallet Foundation's Governing Board?
I ask because I will, in parallel or prior to asking the OWF Board about GPL v3, go to Elia and EW and ask about the possibility of re-licensing to an already known acceptable license.

Reply:
We have not yet gone to the Governing Board for any approvals other than Apache 2.0. So far, each of the proposals has agreed to use Apache 2.0.

Reply:
Hi @jrhender 

I understand that the main point of the License discussion is about [Permissive](https://en.wikipedia.org/wiki/Permissive_software_license#:~:text=Permissive%20licenses%20offer%20more%20extensive,requirements%20conflict%20with%20each%20other.) versus [Copyleft](https://en.wikipedia.org/wiki/Copyleft)

Permissive licenses give more flexibility to libraries and software components to be used by other software products and be combined with other licenses. This is one of the reasons Apache 2.0 is promoted.  

https://fossa.com/blog/open-source-software-licenses-101-gpl-v3/#:~:text=Like%20the%20GPL%20v2%2C%20GPL,released%20under%20the%20GPL%20v3.

Reply:
Thanks for your comment here @tkuhrt and @skounis. Good news: both Energy Web and Elia have agreed to relicense the project as Apache 2.0. We would do so if the project was accepted by the OWF.
Energy Web's copyright is noted in each source file and all of the contributors thus far have been Energy Web and Elia personnel. Therefore, my opinion is that relicensing will be straightforward and low effort in this case.

I've added a commit mentioning the relicensing to the proposal here: https://github.com/openwallet-foundation/project-proposals/pull/12/commits/fec4845ba6a879e8594724de48bbbfa138aa3f0a

Reply:
This is great news, @jrhender.